scrapper.py

```diff
@@ -25,39 +33,94 @@ class Crawler:
     """
     Crawler implementation
     """
+
     def __init__(self, seed_urls, max_articles: int):
-        pass
+        self.seed_urls = seed_urls
+        self.max_articles = max_articles
+        self.urls = []
 
     def _extract_url(self, article_bs):
-        pass
+        for article_link in article_bs.find_all("a", class_="article__download button-icon"):
+            self.urls.append(''.join(["https://journals.kantiana.ru/", article_link.get("href")]))
 
     def find_articles(self):
         """
         Finds articles
         """
-        pass
+
+        headers = {
+            'user-agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 '
+                          '(KHTML, like Gecko) Chrome/99.0.4844.51 Safari/537.36'}
+        sleep_period = random.randrange(3, 7)
+
+        for url in self.seed_urls:
+            response = requests.get(url, headers=headers)
+            if not response.ok:
+                print("Request was unsuccessful.")
```

Comment:
it is a weak check. probably we do not want to process such an URL then? what do you think?

Reply:
Yes, we should skip this url.